packager/react-packager/src/DependencyResolver/__tests__/Module-test.js

```diff
@@ -22,6 +22,7 @@ jest
 var Fastfs = require('../fastfs');
 var Module = require('../Module');
 var ModuleCache = require('../ModuleCache');
+var Helpers = require('../DependencyGraph/Helpers');
```

Comment:
We might want to rename this module to be more descriptive (not sure what though. DependencyGraphHelpers?) since it's being used outside of DependencyGraph now.